app/main.py

```diff
@@ -1,2 +1,97 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity: int = 8
+        self.load_factor: float = 0.75
+        self.size: int = 0
+        self.hash_table: list = [None] * self.capacity
+
+    class Node:
+        def __init__(self, key: str, value: str, hash_code: int) -> None:
```

Comment:
```suggestion
        def __init__(self, key: Hashable, value: Any, hash_code: int) -> None:
```
please fix everywhere